app/timetables/tests/test_models.py

```diff
@@ -99,3 +99,70 @@ def test_duplicate_course_name_cannot_be_saved(self):
         course = Course(name='test')
 
         self.assertRaises(IntegrityError, course.save)
+
+
+class TimetableTest(TestCase):
+    """Tests the Timetable model."""
+
+    def setUp(self):
+        Timetable.objects.create(
+            name='timtable-item',
+            code='FBI23212',
+            api_key='419223',
+            cycle_length=14,
+            current_cycle_day=2,
+            date_created=datetime.time(datetime.strptime('5:7:9', '%H:%M:%S')),
+            date_modified=datetime.time(datetime.strptime('10:7:9', '%H:%M:%S'))
+        )
+
+    def test_duplicate_timetable_name_cannot_be_saved(self):
+        timetable = Timetable(
+            name='timtable-item',
+            code='FB23212',
+            api_key='41923',
+            cycle_length=14,
+            current_cycle_day=2,
+            date_created=datetime.time(datetime.strptime('4:7:9', '%H:%M:%S')),
+            date_modified=datetime.time(datetime.strptime('15:7:9', '%H:%M:%S'))
+        )
+
+        self.assertRaises(IntegrityError, timetable.save)
+
+    def test_duplicate_timetable_code_cannot_be_saved(self):
+        timetable = Timetable(
+            name='timtable-value',
+            code='FBI23212',
+            api_key='41923',
+            cycle_length=14,
+            current_cycle_day=2,
+            date_created=datetime.time(datetime.strptime('4:7:9', '%H:%M:%S')),
+            date_modified=datetime.time(datetime.strptime('15:7:9', '%H:%M:%S'))
+        )
+
+        self.assertRaises(IntegrityError, timetable.save)
+
+    def test_duplicate_timetable_code_cannot_be_saved(self):
```

Comment:
What are we testing for here, it has the same method name as the method above it. Also, I see that the unique fields are different for both test cases, so what are we testing for?